frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -251,6 +260,106 @@ const ManagedNamespaces: React.FC<ManagedNamespacesProps> = ({ obj }) => {
   }
 };
 
+const ConsolePlugins: React.FC<ConsolePluginsProps> = ({ csvPlugins, subscription }) => {
+  const console: WatchK8sResource = {
+    kind: referenceForModel(ConsoleOperatorConfigModel),
+    isList: false,
+    name: CONSOLE_OPERATOR_CONFIG_NAME,
+  };
+  const [consoleOperatorConfig] = useK8sWatchResource<K8sResourceKind>(console);
+  const { t } = useTranslation();
+  const canPatchConsoleOperatorConfig = useAccessReview({
+    group: ConsoleOperatorConfigModel.apiGroup,
+    resource: ConsoleOperatorConfigModel.plural,
+    verb: 'patch',
+    name: CONSOLE_OPERATOR_CONFIG_NAME,
+  });
+  const csvPluginsCount = csvPlugins.length;
+
+  return (
+    <>
+      {consoleOperatorConfig && canPatchConsoleOperatorConfig && (
+        <dl className="co-clusterserviceversion-details__field">
+          <dt>{t('olm~Console UI extension', { count: csvPluginsCount })}</dt>
+          {csvPlugins.map((plugin) => (
+            <dd key={plugin} className="co-clusterserviceversion-details__field-description">
+              {csvPluginsCount > 1 && (
+                <strong className="text-muted">{t('olm~{{plugin}}:', { plugin })} </strong>
+              )}
+              <Button
+                data-test="edit-console-plugin"
+                type="button"
+                isInline
+                onClick={() =>
+                  consolePluginModal({
+                    consoleOperatorConfig,
+                    csvPluginsCount,
+                    plugin,
+                    subscription,
+                  })
+                }
+                variant="link"
+              >
+                <>
+                  {isPluginEnabled(consoleOperatorConfig, plugin)
+                    ? t('olm~Enabled')
+                    : t('olm~Disabled')}{' '}
+                  <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
+                </>
+              </Button>
+            </dd>
+          ))}
+        </dl>
+      )}
+    </>
+  );
+};
+
+const ConsolePluginStatus: React.FC<ConsolePluginStatusProps> = ({ csv, csvPlugins }) => {
+  const console: WatchK8sResource = {
+    kind: referenceForModel(ConsoleOperatorConfigModel),
+    isList: false,
+    name: CONSOLE_OPERATOR_CONFIG_NAME,
+  };
+  const [consoleOperatorConfig] = useK8sWatchResource<K8sResourceKind>(console);
+  const { t } = useTranslation();
+  const canPatchConsoleOperatorConfig = useAccessReview({
+    group: ConsoleOperatorConfigModel.apiGroup,
+    resource: ConsoleOperatorConfigModel.plural,
+    verb: 'patch',
+    name: CONSOLE_OPERATOR_CONFIG_NAME,
+  });
+  const aPluginIsDisabled =
+    !consoleOperatorConfig?.spec?.plugins?.length ||
+    csvPlugins.some((plugin) => isPluginEnabled(consoleOperatorConfig, plugin) === false);
```

Comment:
Can this simply be...?

```suggestion
    csvPlugins.some((plugin) => !isPluginEnabled(consoleOperatorConfig, plugin));
```

I assume `isPluginEnabled` always returns a boolean and never null.

Reply:
Yes.  Good catch.